debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/Filters.java

```diff
@@ -39,37 +39,37 @@
     public Filters(PostgresConnectorConfig config) {
 
         // we always want to exclude PG system schemas as they are never part of logical decoding events
-        String schemaBlacklist = config.schemaBlacklist();
-        if (schemaBlacklist != null) {
-            schemaBlacklist = schemaBlacklist + "," + SYSTEM_SCHEMA_BLACKLIST;
+        String schemaExcludeList = config.schemaExcludeList();
+        if (schemaExcludeList != null) {
+            schemaExcludeList = schemaExcludeList + "," + SYSTEM_SCHEMA_EXCLUDE_LIST;
         }
         else {
-            schemaBlacklist = SYSTEM_SCHEMA_BLACKLIST;
+            schemaExcludeList = SYSTEM_SCHEMA_EXCLUDE_LIST;
         }
 
-        String tableBlacklist = config.tableBlacklist();
-        if (tableBlacklist != null) {
-            tableBlacklist = tableBlacklist + "," + TEMP_TABLE_BLACKLIST;
+        String tableExcludeList = config.tableExcludeList();
```

Comment:
this logis is imo a bit weird and maybe should be fixed. I didn't change the logic to avoid too many unrelated changes.

Reply:
Can you capture this in a Jira issue? There already might be one actually for unifying the include/exclude filter handling. If you find it, can you please add a comment there. There only should be a single `Filters` class (at least for all the relational connectors) to begin with.

Reply:
https://issues.redhat.com/browse/DBZ-2426

Reply:
the global class might be io.debezium.relational.RelationalTableFilters ?

Reply:
Yes, exactly. This one should eventually be the only entry point for logic interested in filtering logic. The biggest obstacle there is the pending refactoring by @jpechane to move MySQL to the common framework. Once that's done, this can be tackled.